dotcom-rendering/src/components/NewsletterPrivacyMessage.tsx

```diff
@@ -25,36 +29,54 @@ const LegalLink = ({ href, children }: LegalLinkProps) => (
 	</Link>
 );
 
-const termsStyle = (textColor: 'supporting' | 'regular') => css`
+const termsStyle = css`
 	${textSans.xxsmall({ lineHeight: 'tight' })}
-	color: ${textColor === 'regular'
-		? palette.neutral[20]
-		: palette.neutral[46]};
 	a {
 		${textSans.xxsmall()};
-		color: ${textColor === 'regular'
-			? palette.neutral[7]
-			: palette.neutral[0]};
 		text-decoration: underline;
 		:hover {
-			color: ${textColor === 'regular'
-				? palette.neutral[7]
-				: palette.neutral[0]};
 			text-decoration: underline;
 		}
 	}
 	strong {
-		color: ${textColor === 'regular'
-			? palette.neutral[7]
-			: palette.neutral[0]};
 		font-weight: bold;
 	}
 `;
+const textStyles = (textColor: 'supporting' | 'regular') => {
+	switch (textColor) {
+		case 'supporting':
+			return css`
+				color: ${sourcePalette.neutral[46]};
+				a {
+					color: ${themePalette('--privacy-text-supporting')};
+					:hover {
+						color: ${themePalette('--privacy-text-supporting')};
+					}
+				}
+				strong {
+					color: ${themePalette('--privacy-text-supporting')};
+				}
```

Comment:
I think this can be slightly simplified:

```suggestion
				a, strong {
					color: ${themePalette('--privacy-text-supporting')};
					:hover {
						color: inherit;
					}
				}
```